custom_components/ha-glocaltokens/api.py

```diff
@@ -46,3 +52,53 @@ def get_google_devices_json(self):
     def get_android_id(self):
         """Generate random android_id"""
         return self._client._get_android_id()
+
+    def get_google_devices_information(self):
+
+        devices = self._client.get_google_devices_json()
+
+        for x in range(len(devices)):
+            #local_token = next(item[GLOCALTOKENS_TOKEN] for item in devices)
+            local_token = devices[0][GLOCALTOKENS_TOKEN] # Else x
+
+            url = 'https://IP OF DEVICE:8443/setup/assistant/alarms' # ONLY FOR TESTING PURPOSE - Until we find a way to retrieve ip of each device.
+            header = {'cast-local-authorization-token': local_token,
```

Comment:
Let's have those headers as constants in the `const.py` file

Reply:
Done